### PR TITLE
Added 5 new thumbnails

### PR DESCRIPTION
**Pull request by ivanlewin**
Added thumbnails for

- 23andMe
- Mercado Libre
- Pluralsight
- Ripio
- Unity

I tested the app and it worked, hope it helps!

### CODE REVIEW

**Comment by ziegenberg** [comment]
I'll have a look during the week. I'm quite busy at the moment

**Comment by flocke** [comment]
No worries, I'm just assigning the review to everybody because it's easier. Whoever finds the time first can do it, even if it takes a while.

File: app/src/main/res/drawable/thumb_23andme.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="136.85dp"
+    android:viewportWidth="200"
+    android:viewportHeight="136.85">
+  <path
+      android:pathData="M10.96,116.7c-1.73,0 -3.39,-0.4 -4.94,-1.18c-2.61,-1.32 -4.56,-3.58 -5.47,-6.37c-0.91,-2.78 -0.69,-5.76 0.64,-8.37l14.69,-29.93l9.36,30.7l-4.49,9.15C18.87,114.39 15.12,116.7 10.96,116.7z"
+      android:fillColor="#7AC043"
+      android:fillType="evenOdd"/>
```
Comment (ziegenberg) [inline app/src/main/res/drawable/thumb_23andme.xml:9]: all occurrences of `android:fillType="evenOdd"` are not necessary and can be removed

File: app/src/main/res/drawable/thumb_pluralsight.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="200dp"
+    android:height="200dp"
+    android:viewportWidth="200"
+    android:viewportHeight="200">
+  <path
+      android:pathData="M99.34,100.29m-100,0a100,100 0,1 1,200 0a100,100 0,1 1,-200 0">
```
Comment (ziegenberg) [inline app/src/main/res/drawable/thumb_pluralsight.xml:8]: The circle is a bit off and peaks out of the viewport. Maybe try `M100,100M0,100a100,100 0,1 1,200 0,100 100,0 1,1 -200,0` for the **pathData** instead.

File: app/src/main/res/drawable/thumb_mercadolibre.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
...c2.07,0 4.09,-0.62 6.03,-1.86c2.47,-1.58 3.96,-4 4.19,-6.82c0.16,-1.92 -0.32,-3.84 -1.33,-5.5c6.68,-2.88 21.96,-8.45 39.95,-12.51c-0.11,-1.4 -0.31,-2.77 -0.56,-4.14C173.8,62.97 157.55,70 153.48,71.81zM92.63,106.99c-4.23,0 -7.66,-3.29 -7.82,-7.48c-0.01,-0.36 -0.05,-1.31 -0.86,-1.31c-0.33,0 -0.62,0.2 -0.95,0.5c-0.93,0.86 -2.11,1.74 -3.84,1.74c-0.78,0 -1.64,-0.18 -2.53,-0.55c-4.46,-1.81 -4.52,-4.88 -4.34,-6.11c0.05,-0.33 0.07,-0.67 -0.16,-0.94l-0.28,-0.25h-0.28c-0.23,0 -0.46,0.09 -0.78,0.32c-1.29,0.91 -2.53,1.35 -3.79,1.35c-0.69,0 -1.4,-0.14 -2.12,-0.41c-5.88,-2.29 -5.41,-7.84 -5.13,-9.5c0.04,-0.34 -0.04,-0.6 -0.26,-0.78l-0.42,-0.34l-0.39,0.37c-1.15,1.1 -2.64,1.71 -4.22,1.71c-3.37,0 -6.1,-2.74 -6.1,-6.1c0,-3.37 2.74,-6.1 6.11,-6.1c3.04,0 5.63,2.29 6.02,5.32l0.21,1.64l0.9,-1.39c0.1,-0.16 2.57,-3.89 7.1,-3.89c0.86,0 1.75,0.14 2.65,0.42c3.62,1.1 4.23,4.38 4.32,5.74c0.06,0.8 0.63,0.84 0.74,0.84c0.31,0 0.54,-0.2 0.7,-0.37c0.68,-0.71 2.17,-1.9 4.49,-1.9c1.06,0 2.2,0.26 3.36,0.76c5.74,2.46 3.14,9.76 3.11,9.84c-0.49,1.21 -0.51,1.74 -0.05,2.05l0.23,0.11h0.17c0.26,0 0.58,-0.11 1.11,-0.29c0.78,-0.27 1.96,-0.68 3.06,-0.67l0,0c4.32,0.05 7.84,3.57 7.84,7.84C100.47,103.48 96.95,106.99 92.63,106.99z"
```
Comment (ziegenberg) [inline app/src/main/res/drawable/thumb_mercadolibre.xml:22]: If it's easy to do, maybe you could split this path in several smaller pieces. With 4770 characters it's rather long and for performance reasons it's recommended to keep it under 800 characters. If it takes too long, we will live with it the way it is and maybe optimize it in the future.

Comment (ivanlewin) [reply app/src/main/res/drawable/thumb_mercadolibre.xml:22]: This one I don't know how to do heh

**Comment by ivanlewin** [comment]
I made 4/5 changes

File: app/src/main/res/drawable/thumb_mercadolibre.xml (hunk 3 above)
Comment (ziegenberg) [reply app/src/main/res/drawable/thumb_mercadolibre.xml:22]: This one is a hard one. We will leave it as is for now.

**Comment by ziegenberg** [comment]
@flocke this is ready for merging. Afterwards I will rebase #573.

**Pull request merged**